src/context-providers/types.ts

```diff
@@ -0,0 +1,28 @@
+import React from "react";
```

Comment:
no need to import react

Reply:
eslint complains if react is not imported